subsys/dfu/dfu_target/src/dfu_target_modem_delta.c

```diff
@@ -23,6 +23,7 @@ struct modem_delta_header {
 };
 
 static dfu_target_callback_t callback;
+int dfu_target_modem_delta_erase(int slot, bool force);
```

Comment:
Is there some specific reason why we use this forward declaration instead of including the `dfu/dfu_target_modem_delta.h` header?

Reply:
The idea was not to have it exposed as a part of a public API but I guess that header is not public so no reason.

Reply:
Well, that header is just as public as `dfu/dfu_target.h`, even if the docs only mention `dfu_target.h`, I can include this header just as easily.